src/EnergyPlus/RootFinder.cc

```diff
@@ -643,21 +643,10 @@ namespace RootFinder {
 
         // Check last as this was not done in the original implementation
         // Essentially we stop the iteration if:
-        // - the increment beween successive iterates is smaller than the user-specified
-        //   tolerance for the X variables.
         // - the distance between the lower and upper bounds is smaller than the user-specified
         //   tolerance for the X variables. (USING brackets from previous iteration)
-        // BUG: Relaxed check to avoid detecting round-off in case 2 successive iterates are the same!
-        // IF ( CheckIncrementRoundOff( RootFinderData, X ) ) THEN
-        //  RootFinderData%StatusFlag = iStatusOKRoundOff
-        //  RETURN
-        // END IF
         if (CheckBracketRoundOff(RootFinderData)) {
             RootFinderData.StatusFlag = iStatusOKRoundOff;
-            RootFinderData.XCandidate = X;
-
-            // Update root finder internal data with current iterate (X,Y)
-            UpdateRootFinder(RootFinderData, X, Y);
```

Comment:
If the upper/lower bracket has been reduced to less than the x tolerance, no need to update the X candidate again, just be done.

Reply:
That makes great sense.